src/stores/proposals.ts

```diff
@@ -5,13 +5,17 @@ import type { Proposal } from '@/types';
 
 type SpaceRecord = {
   loading: boolean;
+  loadingMore: boolean;
   loaded: boolean;
   proposals: Proposal[];
+  hasMoreProposals: boolean;
   summaryLoading: boolean;
   summaryLoaded: boolean;
   summaryProposals: Proposal[];
 };
 
+const PROPOSALS_LIMIT = 20;
```

Comment:
Loading 20 proposals is a bit too much it create large scrollbar can we change to 10?
```suggestion
const PROPOSALS_LIMIT = 10;
```

Reply:
I guess doing just 10 is a bit small for infinite scroll (especially as elements are tiny and with small scroll you would already end up with fetching more).
If we really want to make it lower, maybe we can just lower it for initial load (10), but when fetching more we will fetch 20 again?

Reply:
Even if elements are tiny, with 10 of them it already fit more than a screen page on large screen so I'm not sure it worth loading 20. I don't think we should bother adding more complexity with fetching 10 then 20. But not a strong opinion, I just looked at Twitter and they loading quite a lot of tweet on first load, we can go with 20.